scripts/read_secureboot.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import os
+from os.path import join
+import array
+from fcntl import ioctl
+
+EFIVARS_PATH   = '/sys/firmware/efi/efivars/'
+
+def is_secure_boot_enabled():
+    if not os.path.isdir(EFIVARS_PATH):
+        print("Could not open EFI variables path. Is unable to determine secure boot status")
```

Comment:
Let's not print this. All we should print is, if we determined that SecureBoot is enabled, advice that it is enabled and a recommendation to disable it.

Reply:
Removed it